pkg/provider/azure_test.go

```diff
@@ -1620,6 +1620,13 @@ func getInternalTestService(identifier string, requestedPorts ...int32) v1.Servi
 	svc.Annotations[consts.ServiceAnnotationLoadBalancerInternal] = consts.TrueAnnotationValue
 	return svc
 }
```

Comment:
gofmt the file? there should be an empty line between each functions

Reply:
done